lib/sqlite.core.js

```diff
@@ -501,10 +507,6 @@ SQLitePluginTransaction.prototype.addStatement = function(sql, values, success,
       } else if (t === 'boolean') {
         //Convert true -> 1 / false -> 0
         params.push(~~v);
-      }
```

Comment:
@halaei why did you remove this conditional block? wouldn't it suffice to implement warn function instead...it appears that warn function is the one that is missing...

Reply:
@andpor Maybe I was thinking of backward compatibility or to keep the behavior as strict as possible to minimize possible errors. I don't see any value to silently save `{x: {}}` as "[object Object]". It just makes things more complicated in handling errors. I don't know what will happen when we save symbols or any other types.
Maybe both error and warn was missing, but warn wasn't used anywhere except the code that I removed.
I did this PR a while ago and don't remember all the details. I guess you have already fixed the issue in some other way.